src/SFA.DAS.EmployerAccounts.Web/Views/SearchOrganisation/SearchForOrganisationResults.cshtml

```diff
@@ -18,154 +18,156 @@
+                            </h3>
+                        </legend>
+                        <div class="govuk-radios govuk-radios--small">
+                            <div class="govuk-radios__item">
+                                <input class="govuk-radios__input" id="company" name="organisationType" type="radio" value="CompaniesHouse" @(Model.Data.OrganisationType == OrganisationType.CompaniesHouse ? "checked=\"checked\"" : "")>
+                                <label class="govuk-label govuk-radios__label" for="company">
+                                    Company
+                                </label>
+                            </div>
+                            <div class="govuk-radios__item">
+                                <input class="govuk-radios__input" id="charity" name="organisationType" type="radio" value="Charities" @(Model.Data.OrganisationType == OrganisationType.Charities ? "checked=\"checked\"" : "")>
+                                <label class="govuk-label govuk-radios__label" for="charity">
+                                    Charity
+                                </label>
+                            </div>
+                            <div class="govuk-radios__item">
+                                <input class="govuk-radios__input" id="publicSector" name="organisationType" type="radio" value="PublicBodies" @(Model.Data.OrganisationType == OrganisationType.PublicBodies ? "checked=\"checked\"" : "")>
+                                <label class="govuk-label govuk-radios__label" for="publicSector">
+                                    Public sector
+                                </label>
+                            </div>
+                            <div class="govuk-radios__item">
+                                <input class="govuk-radios__input" id="all" name="organisationType" type="radio" value="" @(Model.Data.OrganisationType == null ? "checked=\"checked\"" : "")>
+                                <label class="govuk-label govuk-radios__label" for="all">
+                                    Show all
+                                </label>
+                            </div>
                         </div>
+                    </fieldset>
+                    <div class="form-group das-js-hide">
+                        <button type="submit" class="govuk-button govuk-button--secondary">Apply filter</button>
                     </div>
-                </fieldset>
-                <div class="form-group">
-                    <button type="submit" class="govuk-button govuk-button--secondary das-visually-hidden">Apply filter</button>
                 </div>
             </div>
-
         </form>
     </section>
 
     <section class="govuk-grid-column-two-thirds">
         @if (Model.Data.Results.Data != null)
         {
-            <ul class="govuk-list das-govuk-list-forms">
-
+            <h2 class="govuk-heading-m">@(Model.Data?.Results.TotalResults ?? 0) results found</h2>
```

Comment:
the  above code been moved from  <span id="inline-search-hint" class="govuk-hint">@(Model.Data?.Results.TotalResults ?? 0) results found</span>  
to here... is the position of the place required for this change.